File: core/src/main/java/fr/sncf/osrd/new_infra_state/implementation/SignalizationError.java
package fr.sncf.osrd.new_infra_state.implementation;

import fr.sncf.osrd.exceptions.OSRDError;
import java.io.Serial;

public class SignalizationError extends OSRDError {

    @Serial
    private static final long serialVersionUID = -4664988804881395290L;
    public static final String osrdErrorType = "signalization";

    /** Constructor*/
    protected SignalizationError(String message, ErrorCause errorCause) {
        super(message, errorCause);
    }
}

File: core/src/main/kotlin/fr/sncf/osrd/dyn_infra/implementation/SignalizationError.kt
package fr.sncf.osrd.dyn_infra.implementation

import fr.sncf.osrd.exceptions.OSRDError
import java.io.Serial

class SignalizationError(message: String?, errorCause: ErrorCause?) : OSRDError(message, errorCause) {
    companion object {
        @Serial
        const val serialVersionUID = -4664988804881395290L
        const val osrdErrorType = "signalization"
    }
}